docs/source/training_vlm_sft.md

```diff
@@ -0,0 +1,386 @@
+# Fine-tuning a Multimodal Model Using SFT (Single or Multi-Image Dataset)
+
+![Single Image + Text](./training_vlm_sft_training_procedure_single_image.png)  
```

Comment:
could you please use https://huggingface.co/datasets/trl-lib/documentation-images for the images? I've just sent you an invite to join the org